cmd/root.go

```diff
@@ -33,7 +34,7 @@ func init() {
 func handleError(err error) {
 	if err != nil {
 		fmt.Println(err)
-		os.Exit(1)
+		panic(err)
```

Comment:
let's switch this back for now. In the future, maybe we have an env `DEBUG=true` or something that would panic instead, in order to show a trace...